README.md

```diff
@@ -300,69 +370,97 @@ Here is a break down of what to expect:
     { id: 'b', name: 'Bob' },
     { id: 'c', name: 'Charlie' }
   ];
-  const collection = new RemoteCollection<User>()
-    .withList('id', users)
+  const collection = new RemoteCollection<User>('id')
+    .withList(users)
     .fetch('d');
 
   assert.deepStrictEqual(
     collection.view(),
     RemoteData.pending
   );
   ```
+  
+Using a view key:
 
-## viewAt
+```ts
+const collection = new RemoteCollection<User>('id')
+  .withList(users, 'team1');
+  
+assert.deepStrictEqual(
+  collection.view(),
+  RemoteData.initial
+);
 
-Like `view` but gets the list that was stored at the given key via `withListAt`,
-`withResourceAt`, `withFailureAt`.
+assert.deepStrictEqual(
+  collection.view('team1'),
+  RemoteData.success(users);
+);
+```
+
+## concat
+
+If you need to append to the list, instead of replacing them, use this method.
+It is very similar to `withList`, and takes an optional view key as the third
+argument.
 
 ### Signature
 ```ts
-public viewAt(at: string): RemoteData<string[], Resource[]>
+concat(
+  resources: Resource[],
+  viewKey?: string = RemoteCollection.DEFAULT_KEY
+): RemoteCollection<Resource>
 ```
 
-## remove
+## map
 
-Removes the resource indicated by the `id`.
+Apply a function to every resource at a view key.
```

Comment:
Just to verify, this `map` function operates in the same way as es6 `map`? E.g. does not mutate the passed in list.

Reply:
That's right. All of the methods return a new `RemoteCollection`, we don't have any mutating methods.

Reply:
Sweet, thanks for the clarification 👍